src/cluster.c

```diff
@@ -5591,6 +5565,41 @@ NULL
                 (retval == C_OK) ? "BUMPED" : "STILL",
                 (unsigned long long) myself->configEpoch);
         addReplySds(c,reply);
+    } else if (!strcasecmp(c->argv[1]->ptr,"slot-stats")) {
+        unsigned char *slots = zmalloc(CLUSTER_SLOTS);
+        memset(slots, 0, CLUSTER_SLOTS);
```

Comment:
This was a direct copy from the [already existing implementations](https://github.com/redis/redis/blob/unstable/src/cluster.c#L5391-L5394) for `CLUSTER ADDSLOTS` and `CLUSTER ADDSLOTSRANGE`.

Question is, why are we using `zmalloc` and `zfree` on a fixed length array? Should we rather define it statically through;
```
unsigned char slots[CLUSTER_SLOTS] = {0);
```